### PR TITLE
Add sign up screen using storyboard

### PR DESCRIPTION
**Pull request by raul0208**
Add sign up screen using Storyboard.

Edited Screenshots:
Screen Shot 2022-09-08 at 12 04 28 PM

### CODE REVIEW

File: ios-target-app/Onboarding/Views/SignUpViewController.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+//  Created by Raul Piñeres Carrera on 6/09/22.
+//
+
+import UIKit
+
+class SignUpViewController: UIViewController {
+
+    @IBOutlet weak var genderTextField: UITextField!
+    
+    @IBOutlet weak var ovalsGroup: UIImageView!
```
Comment (glm4) [inline ios-target-app/Onboarding/Views/SignUpViewController.swift:14]:
```suggestion
    @IBOutlet weak var ovalsImageView: UIImageView!
```

File: ios-target-app/Onboarding/Views/SignUpViewController.swift [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    
+    let genders = ["Female", "Male", "Not defined"]
```
Comment (glm4) [inline ios-target-app/Onboarding/Views/SignUpViewController.swift:16]: are you planning to move the data out of the view controller?

File: ios-target-app/Onboarding/Views/SignUpViewController.swift [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    
+    var genderPickerView = UIPickerView()
```
Comment (glm4) [inline ios-target-app/Onboarding/Views/SignUpViewController.swift:18]: pls make all the UI components properties private

File: ios-target-app/Onboarding/Views/SignUpViewController.swift [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+    /*
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Get the new view controller using segue.destination.
+        // Pass the selected object to the new view controller.
+    }
+    */
```
Comment (glm4) [inline ios-target-app/Onboarding/Views/SignUpViewController.swift:40]:
```suggestion
```

**Review by glm4 — COMMENTED** [review]
@raul0208 I see from the previews that the images are a bit compressed. Also, they should reach to the top of the screen(behind the status bar). These issues are easily fixed from the IB, let me know if you need a hand.

File: ios-target-app/Onboarding/Views/SignUpViewController.swift (hunk 2 above)
Comment (raul0208) [reply ios-target-app/Onboarding/Views/SignUpViewController.swift:16]: Oh not really, I'll make it private then.

**Pull request merged**